pkg/sql/logictest/testdata/logic_test/ranges

```diff
@@ -356,3 +356,45 @@ start_key                          start_pretty                   end_key
 [196 137 246 123]                  /Table/60/1/123                Ċ                                  /Table/60/2                    d              c                 ·           {1}       1
 Ċ                                  /Table/60/2                    [196 138 136]                      /Table/60/2/0                  d              c                 c_i_idx     {1}       1
 [196 138 136]                      /Table/60/2/0                  [255 255]                          /Max                           d              c                 c_i_idx     {1}       1
+
+
+# Due to aynschronous splitting of ranges, we cannot gaurantee the output of the show ranges from database command.
```

Comment:
Please fix misspellings (aynschronous, gaurantee) and limit comment line length to 80 chars when possible.